src/stack.ts

```diff
@@ -152,15 +151,13 @@ export class App
                 const message = e.message as string;
                 const messageParts = message.split('Context lookups have been disabled. ');
                 const missingParts = messageParts[1].split('Missing context keys: ');
-                error(
+                throw new Error(
```

Comment:
There is a control flow difference between before vs after. Before, the `error` command did not terminate the control flow. After, `throw` terminates the control flow. Which one we want here?

Reply:
We want to throw. These errors that are thrown by CDK are fatal errors that prevent synthesis from completing. If we don't terminate here we end up with a very unhelpful error about not being able to read the cloud assembly.